packages/extension-browser/src/manifest.json

```diff
@@ -1,22 +1,22 @@
 {
   "author": "webhint",
   "background": {
-    "scripts": [
+    "service_worker": [
       "background-script.js"
     ],
     "persistent": false
   },
   "icons": {
     "180": "icon.png"
   },
-  "browser_action": {
+  "action": {
       "default_title": "webhint",
       "default_popup": "browser-action/browser-action.html"
   },
   "default_locale": "en",
   "description": "Quickly test your website to see if there are any issues with accessibility, browser compatibility, security, performance and more.",
   "devtools_page": "devtools/devtools.html",
-  "manifest_version": 2,
+  "manifest_version": 3,
   "name": "webhint",
   "permissions": [
     "<all_urls>",
```

Comment:
seems that `<all_urls>`  is not a valid value in `permissions` any more (maybe `host_permissions`)?